R/plumber.R

```diff
@@ -733,7 +740,17 @@ plumber <- R6Class(
           if (!is.null(h$serializer)) {
             res$serializer <- h$serializer
           }
-          req$args <- c(h$getPathParams(path), req$args)
+          parsers <-
+            if (!is.null(h$parsers)) {
+              h$parsers
+            } else {
+              make_parser(private$default_parsers)
```

Comment:
making parsers on each call to the API from a character vector could lower performance, consider pre-making these.

Reply:
Having `make_parser(foo)` automatically return if already parsed. So this works: `make_parser(make_parser(foo))`

Reply:
Just saw you changed it in initialize, it is all good.

Reply:
😄 It was a good catch! Spot on.